### PR TITLE
hooli-style-popup

### PR DESCRIPTION
**Pull request by annymax**
### HTML-CSS-Practice: Hooli-style-popup

[Demo](https://annymax.github.io/HTML-CSS-Practice/) |
[Code base](https://github.com/annymax/HTML-CSS-Practice)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by annymax** [comment]
I have completed self-checks  and fixed code. Please, review.

File: submissions/annymax/hooli-style-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+          </li>
+          <li class="nav-list-item">
+            <a class="nav-link" href="">Images</a>
+          </li>
+          <li class="nav-list-item">
+            <input class="input-popup" type="checkbox" id="button-popup" />
+            <label class="label-popup" for="button-popup">
+              <img src="icons/popup-button.png" alt="" class="img-popup" />
+            </label>
+            <div class="menu-wrapper">
```
Comment (A-Ostrovnyy) [inline submissions/annymax/hooli-style-popup/index.html:25]: Please fix the size of your popup it's hard to read the content inside.
image

File: submissions/annymax/hooli-style-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+.bell,
+.favicon,
+.img-popup {
+  display: block;
+  width: 27px;
+  height: 27px;
+  box-sizing: border-box;
+}
+
+::-webkit-scrollbar {
```
Comment (A-Ostrovnyy) [inline submissions/annymax/hooli-style-popup/style.css:184]: It's an invalid selector. You can't take a pseudo-element from nothing. [Read about it](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Sorry for the late response.

File: submissions/annymax/hooli-style-popup/index.html (hunk 1 above)
Comment (annymax) [reply submissions/annymax/hooli-style-popup/index.html]: made bigger size of content

File: submissions/annymax/hooli-style-popup/style.css (hunk 2 above)
Comment (annymax) [reply submissions/annymax/hooli-style-popup/style.css]: added class for scrollbar (  if you mean it )

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**